flytekit/remote/remote.py

```diff
@@ -75,6 +76,14 @@
 MOST_RECENT_FIRST = admin_common_models.Sort("created_at", admin_common_models.Sort.Direction.DESCENDING)
 
 
+def ignore_extra_print(*args, **kwargs):
+    if "fg" in kwargs:
+        del kwargs["fg"]
+    if "nl" in kwargs:
+        del kwargs["nl"]
+    print(*args, **kwargs)
```

Comment:
can we add context for this helper function? not sure what "fg" and "nl" are

Reply:
sure, they are to support click.secho. we need to find a nice way to integrate cli output and logs.

Reply:
i was hoping to use rich logging?